File: src/main/java/no/nav/syfo/domain/Gyldighetstidspunkt.java
package no.nav.syfo.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDate;

@Data
@Accessors(fluent = true)
public class Gyldighetstidspunkt {
    public LocalDate fom;
    public LocalDate tom;
    public LocalDate evalueres;
}

File: src/main/kotlin/no/nav/syfo/api/v2/mapper/Gyldighetstidspunkt.kt
package no.nav.syfo.api.v2.mapper

import no.nav.syfo.api.v2.domain.oppfolgingsplan.Gyldighetstidspunkt

fun Gyldighetstidspunkt.toGyldighetstidspunkt(): no.nav.syfo.domain.Gyldighetstidspunkt {
    val gyldighetstidspunkt = no.nav.syfo.domain.Gyldighetstidspunkt()
    gyldighetstidspunkt.fom = fom
    gyldighetstidspunkt.tom = tom
    gyldighetstidspunkt.evalueres = evalueres
    return gyldighetstidspunkt
}
